stor/cli.py

```diff
@@ -268,6 +268,12 @@ def get_path(pth, mode=None):
     return prefix / path_part.split(rel_part, depth)[depth].lstrip('/')
 
 
+def _to_url(path):
+    if stor.is_filesystem_path(path):
+        raise ValueError('must be swift or s3 path')
```

Comment:
You should be able to make valid urls out of filesystem paths too, right?
```
curl file://localhost/<path>
```

Reply:
oh you know that's a really good point, then you can always do things like `open $(stor url $PATH)` will change

Reply:
okay, I'm actually only going to implement this for linux, because I'm not sure how it ought to work on windows :(